google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionOptions.java

```diff
@@ -231,6 +234,10 @@ public String[] getValidValues() {
                       READONLY_PROPERTY_NAME,
                       "Should the connection start in read-only mode (true/false)",
                       DEFAULT_READONLY),
+                  ConnectionProperty.createBooleanProperty(
+                      ROUTE_TO_LEADER_PROPERTY_NAME,
+                      "Should the RW/PDML requests be routed to leader region (true/false)",
```

Comment:
nit: This string is allowed to be long, so I would suggest the following additions:

```suggestion
                      "Should read/write transactions and partitioned DML be routed to leader region (true/false)",
```